src/styles/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  padding: 32px 16px 16px;
+
+  width: 200px;
+  margin: 0;
+
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  background: $fone-color;
+
+  &__items {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+  }
+
+  &__icon {
+    display: flex;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    display: flex;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $text-main-color;
+
+    margin: auto;
+    margin-bottom: 4px;
+  }
+
+  &__product {
+    margin: 0 0 16px;
+    line-height: 14px;
+
+    color: $product-price-color;
+    font-size: 10px;
+  }
+
+  &__star {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__review {
+    margin: 0;
+
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price--name {
+    display: flex;
+    margin: 0;
+
+    color: $product-price-color;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price--amount {
+    margin: 0;
+
+    color: $text-main-color;
+    font-weight: 700;
+    line-height: 18px;
+    font-size: 16px;
+  }
+
+  &__button {
+    display: flex;
+    width: 100%;
+    align-items: center;
+    padding: 12px;
+
+    color: $fone-color;
+    justify-content: center;
+    font-weight: 700;
+    font-size: 14px;
+
+    border-radius: 5px;
+    background: $button-color;
+
+    &:hover {
+      border-radius: 5px;
+      border: 1px solid $button-color;
```

Comment:
```suggestion
      border-color: $button-color;
```

Reply:
this is was really surprise for me!
why we should delete **1px solid**?
in the figma I saw border-radius)

After all changes everything is nice. But is really good question for me still !
why we delete border-radius?

@Viktor-Kost 

Thanks for your help!

Reply:
since you've already set border-radius: 5px; for your button. And at hover effect this rule isn't changing, so no needs to set it again